[synthetics-command] Add DATADOG_LOCATIONS env variable to override locations (#365)
DataDog/datadog-ci: src/commands/synthetics/README.md

export DATADOG_SUBDOMAIN="myorg"
```

> You can use the `DATADOG_SYNTHETICS_LOCATIONS` to override the locations where your tests run. Locations should be separated with `;`. Note that the configuration of the test will always the precedent over others overrides.  

Reviewer: ```suggestion
You can use the `DATADOG_SYNTHETICS_LOCATIONS` to override the locations where your tests run. Locations should be separated with `;`. Note that the configuration in test files takes precedence over others overrides.  
```
